recipes/ensembl-vep/meta.yaml

```diff
@@ -8,21 +8,21 @@ source:
   md5: 2c088ead3a3a5245a7209e9b087666af
 
 build:
-  number: 1
+  number: 2
   skip: True # [osx]
 
 requirements:
   build:
     - curl
+    - perl
     - perl-archive-zip
     - perl-lwp-simple
-    - perl-threaded >=5.2*
     - unzip
-    - zlib
   run:
-    - htslib 1.3.*
+    - perl
+    - htslib {{CONDA_HTSLIB}}*
```

Comment:
Thanks for the heads up, and sorry for missing this. I have a new PR fixing the recipe build strings.